Automattic/newspack-plugin
# Reader Revenue: Rework "welcome" screen to use ActionCard/Plugin Toggle

### All Submissions:






### Changes proposed in this Pull Request:

This PR aims to simplify the Reader Revenue wizard and list all the different services we offer.

* Some services are mandatory so it's using an ActionCard
* Some services are optional, like LaterPay, and are using the PluginToggle

This will allow us to easily extend the list of services we support.

See p2-pamTN9-L2 for more info.

Update: I have merged Memberships with Donations since the two were similar.

Todo (and need help -- cc @jeffersonrabb 👋 )

* [x] Hide the TabbedNavigation when visiting the Donations and Memberships pages.
* [x] Add a "Back to Monetization Services" button to the Donations and Memberships pages.

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check the Reader revenue wizard page

### Other information:
